product_docs/docs/epas/12/epas_compat_tools_guide/index.mdx

```diff
@@ -28,7 +28,7 @@ The EDB\*Plus command line client provides a user interface to Advanced Server t
 -   Execute OS commands
 -   Record output
 
-For detailed installation and usage information about EDB\*Plus, see the EDB\*Plus User's Guide, available from the EnterpriseDB website at:
+For detailed installation and usage information about EDB\*Plus, see the *EDB\*Plus User's Guide*, available from the EnterpriseDB website at:
 
 [https://www.enterprisedb.com/docs/p/edbplus](/epas/latest/edb_plus/)
```

Comment:
Remove - this link isn't in the correct format and would result in a 404 error. See earlier comment for the preferred format and correct URL.

Reply:
Updated the URL's and checked-in, pls verify and merge. Thanks